test-dssspell.sh

```diff
@@ -20,14 +20,16 @@ if [[ -z "$OPTIMIZER" ]]; then
   OPTIMIZER=1
 fi
 
-export DAPP_BUILD_OPTIMIZE="$OPTIMIZER"
-export DAPP_BUILD_OPTIMIZE_RUNS=1
+# 2022-01-28 Disabled optimizer on due to engineering costs
```

Comment:
You don't need to disable this. Just use `make test-dev` to turn optimizations off.

Reply:
I can fix this, I just revert the prev commit

Reply:
oh well it is done because now we want always optimizations off

Reply:
nvm we're keeping optimizations off with this change